packages/contracts/contracts/Greenproof.sol

```diff
@@ -28,6 +28,8 @@ contract Greenproof is SolidStateDiamond {
      * @custom:field revokerRole - Credential role name granting revoker rights
      * @custom:field workerRole - Credential role name allowing voters to be whitelisted and authorized in the voting system
      * @custom:field claimerRole - Credential role name allowing to claim a certificate on the behalf of others
+     * @custom:field approverRole - Credential role name allowing to set certificate transfer approvals on the behalf of others
+     * @custom:field transferRole - Credential role name allowing to transfer certificates on the behalf of others
```

Comment:
this is not needed anymore :)

Reply:
fixed on commit [8e3e77a](https://github.com/energywebfoundation/worker-contract/pull/186/commits/8e3e77a2c0d95101affc4fde20e0bdaee4068544)